### PR TITLE
GraphQL: StoreConfig attribute updates

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) updates the StoreConfig objects with additions and deprecations added in internal ticket MC-39744.

## Affected DevDocs pages
-  https://devdocs.magento.com/guides/v2.4/graphql/queries/available-stores.html
- https://devdocs.magento.com/guides/v2.4/graphql/queries/store-config.html

## Links to Magento source code

-  ...

### CODE REVIEW

**Review by erikmarr — APPROVED** [review]

File: src/_includes/graphql/store-config.md [hunk 1]
```diff
@@ -80,12 +82,18 @@ Attribute | Data Type | Description | Default or example value
...
+`store_name` | String | The label assigned to the store view | Default Store View
+`store_sort_order` | Int | The store view sort order | 10
 `timezone` | String | The store's time zone | `America/Chicago`
 `title_prefix` | String | A prefix that appears before the title to create a two- or three-part title | null
 `title_separator` | String | Identifies the character that separates the category name and subcategory in the browser title bar | `-`
 `title_suffix` | String | A suffix that appears after the title to create a two-or three part title | null
-`website_id` | Integer | The ID number assigned to the parent website | `1`
+`website_code` | ID | The unique ID for the website | `base`
+`website_id` | Integer | Deprecated. The field should not be used on the storefront. The ID number assigned to the parent website | `1`
+`website_name` | ID | The label assigned to the website | Main Website
```
Comment (rogyar) [inline src/_includes/graphql/store-config.md:96]: Hi @keharper. I just wanted to double-check that the `website_name` has the `ID` type. From a technical standpoint, it's supposed to be `String`.

Thanks.

Comment (keharper) [reply src/_includes/graphql/store-config.md:96]: @rogyar Ugh. That's how it's defined in the schema file. I've notified Cris. Hopefully, we can change this before 2.4.2 ships

Comment (keharper) [reply src/_includes/graphql/store-config.md]: Good news. We're fixing it.

**Review by cpartica — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**